File: 10.js
function oneToOnehundredFor() {
  for (let i = 1; i <= 100; i++) {
    console.log(i);
  }
}

function oneToOnehundredWhile() {
  let i = 1;
  while (i <= 100) {
    console.log(i);
    i++;
  }
}

function oneToOnehundredDoWhile() {
  let i = 1;
  do {
    console.log(i);
    i++;
  } while (i <= 100);
}

function oneToOnehundredForOf() {
  for (let i of Array.from({ length: 100 }, (_, i) => i + 1)) {
    console.log(i);
  }
}

File: 10.py

def one_to_onehundred_for():
    for i in range(1, 101):
        print(i)


def one_to_onehundred_while():
    i = 1
    while i <= 100:
        print(i)
        i += 1


def one_to_onehundred_recursive():
    def print_number(n):
        if n > 100:
            return
        print(n)
        print_number(n + 1)
    print_number(1)
